README.md

```diff
@@ -49,7 +49,7 @@ On Linux:
 ```console
 curl -Lo ./kind "https://kind.sigs.k8s.io/dl/v0.14.0/kind-$(uname)-amd64"
 chmod +x ./kind
-mv ./kind /some-dir-in-your-PATH/kind
+mv ./kind /usr/local/bin/kind
```

Comment:
this path is going to require root permissions typically, so probably make it:
```suggestion
sudo mv ./kind /usr/local/bin/kind
```

?

Reply:
It may even be worth replacing both line 51 and 52 with this:

```
sudo install -o root -g root -m 0755 ./kind /usr/local/bin/kind
```

Reply:
@BenTheElder Thanks for pointing it out. My bad, I missed it. It's fixed now!